10_onion_routing.asciidoc

```diff
@@ -331,6 +331,26 @@ Thus, they will both get the same result which they can use as a shared key to s
 
 ****
 
+A unique trait of Sphinx as a mix-net packet format is that rather than include a distinct session key for each hop in the route, which would increase the size of the mix-net packet dramatically, instead a clever _blinding_ scheme is used deterministically randomize the session key at each hop.
+
+In practice, this little trick allows us to keep the onion packet as compact as possible while still retaining the desired security properties.
+
+The session key for hop `i` is derived using the node public key, and derived shared secret of hop `i-1`:
+```
+session_key_i = session_key_{i-1} * SHA256(node_pubkey_{i-1} || shared_secret_{i-1})
+```
+
+In other words, we take the session key of the prior hop, and multiply it by a value derived from the public key and derived shared secret for that hop.
+
+As EC multiplication can be performed on a public key without knowledge of the private key, each hop is able to re-randomize the session key for the next hop in a deterministic fashion.
+
+The creator of the onion packet knows all the shared secrets (as they're encrypted the packet uniquely for each hop), and thus is able to derive all the blinding factors.
```

Comment:
Edit for clarity (plural verb form for singular 'they'):
"The creator of the onion packet knows all the shared secrets (as they've encrypted the packet uniquely for each hop), and thus i̶s̶ **are** able to derive all the blinding factors."

Reference: https://apastyle.apa.org/style-grammar-guidelines/grammar/singular-they

Reply:
fixed!